book/website/collaboration/new-community/new-community-onboarding.md

```diff
@@ -0,0 +1,98 @@
+(cl-new-community-onboarding)=
+# Virtual Onboarding of New Members
+
+## Summary
+
+Joining a new group virtually is a very different experience from joining in person, and it can be hard to know if someone is struggling or isolated.  
+You can help to make this experience less overwhelming by having a clear onboarding plan and resources. 
+
+## Introducing the Team
+
+It can be intimidating to be the first to speak as a new member - consider asking your team to do a "who's who", and to introduce themselves first in meetings.
+
+Names and roles are also easy to forget, particularly if we aren't seeing people in the office every day. 
+It can be helpful to store this information somewhere, for example, by sending around an introductory email.
+Also consider organising online social events, such as coffee chats or team activity like a video game or escape room to help new members integrate into the group. 
+
+An onboarding event sharing important information like contact details, organigram, shared calendar, resources available to everyone, organisation policy and social norms can be extremely useful.
+
+## Support
+
+You can provide one-on-one support - such as a buddy for informal peer support, or a mentor, for more formal support.
+This helps the new joiner understand group norms, and allows them to ask any sensitive questions they wouldn't like to raise in a group discussion.
+
+## Onboarding Document Guidelines
+
+Your project can have an onboarding document or manual. 
+This creates a set of shared expectations and acts as a reference for new members to find information about resources, or who to ask for help if they have a problem.
+
+### Suggested Sections and Content
+
+#### Group Culture
+
+Every group or workplace has a different culture, and expectations aren't always clear when a new member joins. 
+You can help them settle in by making the culture clear here. 
+
+This can be especially important for people who experience discrimination - do they know that they can approach you and be listened to if they experience harassment or discrimination?
+
+Group culture sections can include:
+* What members of the group can expect from you - turnaround times for feedback, support.
```

Comment:
Who do you mean by you? Leadership group? The project lead? Line managers? I think this needs to be made more clear